### ISSUE TITLE
dosvox.py AppModule should suspend keyboard handling

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#263]
**Reported by clev on 27 Sep 2008 15:17 UTC**
**Blocked by #184**
Thanks Mike for adding dosvox.py to NVDA. It should be noted that Dosvox is made of several applications, such as an editor called Edivox, a browser called webvox, etc. Thus, for NVDA to work properly with, say, Edivox, the user must either save dosvox.py as edivox.py or make a copy of dosvox.py and rename it to edivox.py. I've added this explanation to dosvox.py and I'm now attaching it again.

In addition, besides loosing speech, I'd like NVDA to loose control of the keyboard whenever a Dosvox application gains focus, because sometimes Dosvox uses the insert and capsLock keys. Can you implement this please?

Many thanks

**Comment by nvdaTrac** [comment]
**Comment by jteh on 18 May 2009 07:32 UTC**
Done in r2854.
